packages/spacetime_lib/spacetime_lib.0.3.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+synopsis: "Library for decoding OCaml spacetime profiles"
+maintainer: "Leo White <leo@lpw25.net>"
+authors: [
+  "Leo White <leo@lpw25.net>" "Mark Shinwell <mshinwell@janestreet.com>"
+]
+license: "MIT"
+homepage: "https://github.com/lpw25/spacetime_lib"
+bug-reports: "https://github.com/lpw25/spacetime_lib"
+depends: [
+  "ocaml"
+  "dune" {build & >= "1.0"}
+  "ocamlfind" {build}
```

Comment:
```suggestion
```
I don't see any use of ocamlfind nor findlib

Reply:
Well spotted